components/automate-ui/src/app/modules/infra-proxy/org-details/org-details.component.ts

```diff
@@ -117,6 +105,8 @@ export class OrgDetailsComponent implements OnInit, OnDestroy {
     switch (tab) {
       case 0:
         this.telemetryService.track('orgDetailsTab', 'cookbooks');
```

Comment:
Suggest make this a local constant since it is used a lot:
```suggestion
        this.telemetryService.track(ORG_DETAILS_TAB_NAME, 'cookbooks');
```